modules/elements-color-picker/assets/js/editor/commands/end.js

```diff
@@ -0,0 +1,26 @@
+import CommandBase from 'elementor-api/modules/command-base';
+
+export class End extends CommandBase {
+	apply( args ) {
+		elementor.$previewContents.find( 'body' ).removeClass( 'elementor-editor__ui-state__color-picker' );
+
+		elementor.$previewContents.find( '.elementor-element-color-picker' ).remove();
+
+		elementor.$previewContents.off( 'mouseenter.color-picker' );
+
+		elementor.$previewWrapper.off( 'mouseleave.color-picker' );
```

Comment:
```suggestion
		elementor.$previewContents.off( 'mouseenter.color-picker mouseleave.color-picker' );
```

Reply:
It's not the same element (`$preview**Wrapper**` && `$preview**Contents**`).

Reply:
Opss